javascript/ql/src/Security/CWE-078/CommandInjection.inc.qhelp

```diff
@@ -2,43 +2,63 @@
   "-//Semmle//qhelp//EN"
   "qhelp.dtd">
 <qhelp>
-<overview>
-<p>Code that passes user input directly to
-<code>require('child_process').exec</code>, or some other library
-routine that executes a command, allows the user to execute malicious
-code.</p>
 
+<overview>
+<p>Code that passes untrusted user input directly to
+<code>child_process.exec</code> or similar APIs that execute commands by
+spawning a shell allows the user to execute malicious code.</p>
 </overview>
+
 <recommendation>
+<p>If possible, use hard-coded string literals to specify the command to run.
+Instead of interpreting the user input directly as a shell command, examine the
+user input and then choose among hard-coded string literals.</p>
```

Comment:
I think this advise is rarely applicable.  

I think we should focus on the safe-API/sanitization recommendations (in that order).

Reply:
I agree. I believe it was copy-pasted from the query help for other languages a long time ago. Happy to remove.